src/LogicItLab/Salesforce/MapperBundle/WsdlValidator.php

```diff
@@ -12,41 +11,30 @@ class WsdlValidator
     private $annotationReader;
 
     /** @var string */
-    private $srcPath;
+    private $baseProjectDir;
 
     /** @var string */
-    private $relativePath;
+    private $modelDirPath;
 
     /** @var string */
     private $wsdlPath;
 
+    /** @var string */
+    private $wsdlContents;
+
     /**
      * @param AnnotationReader $annotationReader
-     * @param string $srcPath
-     * @param string $relativePath
-     * @param string $wsdlPath
      * @codeCoverageIgnore
      */
-    public function __construct(AnnotationReader $annotationReader, string $srcPath, string $relativePath, string $wsdlPath)
+    public function __construct(AnnotationReader $annotationReader)
     {
         $this->annotationReader = $annotationReader;
-        $this->srcPath = $srcPath;
```

Comment:
No sure why these arguments have been removed. 
1. The class will not do what it is meant to do if you forget to set them on a new instance.
2. There are no checks to see if these values are set and throw an exception (if you really want to do it this way).